src/Serializer/SerializerContextBuilder.php

```diff
@@ -52,12 +52,19 @@ public function createFromRequest(Request $request, bool $normalization, array $
         } elseif (isset($attributes['item_operation_name'])) {
             $operationKey = 'item_operation_name';
             $operationType = OperationType::ITEM;
+        } elseif (isset($attributes['resource_operation_name'])) {
+            $operationKey = 'resource_operation_name';
+            $operationType = 'resource';
         } else {
             $operationKey = 'subresource_operation_name';
             $operationType = OperationType::SUBRESOURCE;
         }
 
-        $context = $resourceMetadata->getTypedOperationAttribute($operationType, $attributes[$operationKey], $key, [], true);
+        if ('resource' !== $operationType) {
+            $context = $resourceMetadata->getTypedOperationAttribute($operationType, $attributes[$operationKey], $key, [], true);
```

Comment:
It would be better to add support the new type in `getTypedOperationAttribute()`.

Reply:
Like the previous comment, I'm not sure it's a good idea to do this, since it's a "special" operation (for instance the resource fallback parameter doesn't make sense).
I've changed it anyway.